breakdance.lua

```diff
@@ -0,0 +1,13 @@
+--Breaks up a stuck dance activity.
```

Comment:
Maybe `stop-dance` for clarity.

Reply:
Not even sure if this one is necessary actually, probably just pull it out entirely since it was mostly a workaround for the bug that caused hung dances which I think Toady fixed.